app/assets/javascripts/veterans.js.coffee

```diff
@@ -9,6 +9,7 @@ $ ->
   osmUrl = '//{s}.tile.openstreetmap.org/{z}/{x}/{y}.png'
   osmAttrib = 'Map data © <a href="http://openstreetmap.org">'
   osmAttrib += 'OpenStreetMap</a> contributors'
+  markers = L.markerClusterGroup()
```

Comment:
@walleyyang Should set a lower `maxClusterRadius` option on the `L.markerClusterGroup`? 

The default is 80, and at the map zoom of 4 that we're using, sometimes markers that are several states apart are clustered together (for example, I added one marker in zip 40160 - near Louisville KY, and another at zip 30301 - Atlanta, GA, and it clustered them together).  I think that limits the value of the map/clustering. I think the clustering adds value when clustering together markers that are near each other, for example, two markers in the NYC metro area, or several markers near each other in one state.  But if they're several states apart (a 6 hour drive in the KY - ATL example), not so much at the default map zoom level we're starting at.

Decreasing the max cluster radius would allow more, smaller clusters, which I think would be useful.